build/ci/github_actions/driver/deploy_driver.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -xel
+set +o pipefail
+
+expected_node_image=$node_repository_for_test:$driver_images_tag
+expected_controller_image=$controller_repository_for_test:$driver_images_tag
+
+install_worker_prerequisites() {
+  kind_node_name=`docker ps --format "{{.Names}}"`
+  docker exec -i $kind_node_name apt-get update
+  docker exec -i $kind_node_name apt -y install open-iscsi
+}
+
+edit_cr_images (){
+  chmod 547 $cr_yaml
+  declare -A cr_image_fields=(
+      [".spec.controller.repository"]="$controller_repository_for_test"
+      [".spec.controller.tag"]="$driver_images_tag"
+      [".spec.node.repository"]="$node_repository_for_test"
+      [".spec.node.tag"]="$driver_images_tag"
+  )
+  for image_field in ${!cr_image_fields[@]}; do
+      cr_image_value=${cr_image_fields[${image_field}]}
+      yq eval "${image_field} |= \"${cr_image_value}\"" $cr_yaml -i
+  done
+}
+
+install_worker_prerequisites
+edit_cr_images
+cat $cr_yaml | grep repository:
+cat $cr_yaml | grep tag:
+kubectl apply -f $cr_yaml
+. build/ci/github_actions/deployment.sh
```

Comment:
I think source is more readable, but either way, [choose one](https://github.com/IBM/ibm-block-csi-driver/blob/3b3c0b9a16f14e4d49be9aa59bd56c9beb8f4881/scripts/ci/github_actions/driver/setup_ci_dependencies.sh#L9) and stick to it

Reply:
done (with source)